frontend/src/features/frame/components/topbar/AccountInfo.tsx

```diff
@@ -78,8 +83,10 @@ const LoggedOutActions = () => {
   return (
     <>
       <Button
-        component={Link}
         variant="outlined"
+        color="inherit"
+        style={{ borderColor: 'rgba(0, 0, 0, 0.23)' }}
```

Comment:
If possible, we could use `sx` instead of `style` to prevent the generation of the `style` attribute in HTML.

In the future, we might be able to completely disable inline style in the content security policy, if an only if, Material UI drops the usage of `style` in their own components. Who knows?

```suggestion
        sx={{ borderColor: 'rgba(0, 0, 0, 0.23)' }}
```

Reply:
Oops, sorry @amatissart already suggested that change.

Reply:
Fixed in ffae93a